server/auth/sso/sso.go

```diff
@@ -139,10 +140,13 @@ func newSso(
 	// if it fails, then the get will fail, and the pod restart
 	// it may fail due to race condition with another pod - which is fine,
 	// when it restart it'll get the new key
-	_, _ = secretsIf.Create(&apiv1.Secret{
+	_, err = secretsIf.Create(&apiv1.Secret{
```

Comment:
RBAC should have 'create` permission. right?

Reply:
correct - that is already the case - this surfaces issue when users have their own custom RBAC